test/freetype_test.py

```diff
@@ -1794,6 +1794,14 @@ def test_get_error(self):
 
         self.assertIsNone(error_msg)
 
+    def test_get_version(self):
+        # Test that get_version() can be called before init()
+        # Also tests that the compiled and linked versions are
+        # the same
+        ft.quit()
+
+        self.assertEquals(ft.get_version(linked=False), ft.get_version(linked=True))
```

Comment:
The compiled and linked versions could potentially be different, and that would be fine. We’ve seen this with SDL at least— people have had different compiled and linked versions, as long as the linked version is more recent.